Kernel/Prekernel/Arch/aarch64/GPIO.cpp

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2021, Nico Weber <thakis@chromium.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <Kernel/Prekernel/Arch/aarch64/GPIO.h>
+#include <Kernel/Prekernel/Arch/aarch64/MMIO.h>
+
+extern "C" void wait_cycles(int n);
+
+namespace Prekernel {
+
+// See BCM2835-ARM-Peripherals.pdf section "6 General Purpuse I/O".
```

Comment:
```suggestion
// See BCM2835-ARM-Peripherals.pdf section "6 General Purpose I/O (GPIO)".
```

They spell it correctly, I checked :^)